vivisect/tests/linux_arm_sh_data.py

```diff
@@ -957,7 +957,7 @@
         (0x11de0, u'sh.ptr___environ_00011de0'),
         (0x12010, u'sh.ptr_plt_mkstemp_00012010'),
         (0x12014, u'sh.ptr_stderr_00012014'),
-        (0x122d0, u'sh.ptr_plt_mkstemp_000122d0'),
+        # (0x122d0, u'sh.ptr_plt_mkstemp_000122d0'),
```

Comment:
i've made some changes of my own that should wipe these extra's out (in switchcase), this is good for now.